src/client/src/ui/spotTile/model/rfqRequest.ts

```diff
@@ -0,0 +1,33 @@
+import { CurrencyPair } from '../../../rt-types/currencyPair'
+import { SpotPriceTick } from './spotPriceTick'
+
+export interface RfqRequest {
+  notional: number
+  currencyPair: CurrencyPair
+}
+
+export interface RfqReceived extends RfqRequest {
+  price: SpotPriceTick
+  timeout: number
+}
+
+export interface RfqRequote {
+  notional: number
+  currencyPair: CurrencyPair
+}
+
+export interface RfqReject {
```

Comment:
Suggestion: We could extend an interface that has a CurrencyPair

Reply:
`tslint` would complain.
<img width="1247" alt="Screen Shot 2019-04-03 at 10 18 55 AM" src="https://user-images.githubusercontent.com/1565793/55492646-8d7b3d00-5605-11e9-9b41-657a6067e23e.png">

I could of used a single type that's for sure but I figured we might as well prevent refactors as these might evolve (add timestamp, etc) and it would be more readable in the code.